docs/linux.md

```diff
@@ -84,6 +84,57 @@ PYTHON_BINARY, NODE_BINARY, YARN_BINARY and/or PROTOC_BINARY to use locally-inst
 If rust-toolchain.toml is removed, newer Rust versions can be used. Older versions
 may or may not compile the code.
 
+To build Anki fully offline, set the following environment variables:
+
+- OFFLINE_BUILD: If set, the build does not run tools that may access
+  the network.
+
+- NODE_BINARY, YARN_BINARY and PROTOC_BINARY must also be set.
+
+With OFFLINE_BUILD defined, manual intervention is required for the
+offline build to succeed. The following conditions must be met:
+
+1. All required dependencies (node, Python, rust, yarn, etc.) must be
+   present in the build environment.
+
+2. The offline repositories for the translation files must be
+   copied/linked to ftl/qt-repo and ftl/core-repo.
+
+3. The Python pseudo venv must be set up:
+
+   ```
+   mkdir out/pyenv/bin
+   ln -s /path/to/python out/pyenv/bin/python
+   ln -s /path/to/protoc-gen-mypy out/pyenv/bin/protoc-gen-mypy
+   ```
+
+   Optionally, set up your environment to generate Sphinx documentation:
+
+   ```
+   ln -s /path/to/sphinx-apidoc out/pyenv/bin/sphinx-apidoc
+   ln -s /path/to/sphinx-build out/pyenv/bin/sphinx-build
+   ```
+
+   Note that the PYTHON_BINARY environment variable need not be set,
+   since it is only used when OFFLINE_BUILD is unset to automatically
+   create a network-dependent Python venv.
+
+4. Create the offline cache for yarn and use its own environment
+   variable YARN_CACHE_FOLDER to it:
+
+   ```
+   YARN_CACHE_FOLDER=/path/to/the/yarn/cache
+   /path/to/yarn install --ignore-scripts
+   ```
+
+5. Build Anki:
+
+   ```
+   /path/to/cargo build --package runner --release
+   OFFLINE_BUILD=1 \
+   out/rust/release/runner build wheels
+   ```
+
```

Comment:
Maybe it makes sense to add here an additional point, e.g. 

6. Build documentation with Sphinx (optional)
```
out/rust/release/runner build python:sphinx
```

Reply:
How about something like this instead?
```
You are now ready to build wheels and Sphinx documentation fully
offline.
```
I think, I should try not to duplicate instructions on how to build wheels and Sphinx documentation. Both of these are already describe in `docs/development.md` and this file is mentioned in the very next section.
Also, if anything, step 5 should probably read `OFFLINE_BUILD=1 ./ninja ...` since no other documented installation command exposes the build systems internals.

Reply:
Makes sense and your suggestion looks fine for me.

Reply:
Thanks for your input!